src/install/kafka/whatsNext.mdx

```diff
@@ -0,0 +1,4224 @@
+
+      <th>
+        Default
+      </th>
+
+      <th>
+        Applies To
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        **KEY_STORE**
+      </td>
+
+      <td>
+        The filepath of the keystore containing the JMX client's SSL certificate.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ "text-align": "center" }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **KEY_STORE_PASSWORD**
+      </td>
+
+      <td>
+        The password for the JMX SSL key store.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **TRUST_STORE**
+      </td>
+
+      <td>
+        The filepath of the trust keystore containing the JMX server's SSL
+        certificate.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **TRUST_STORE_PASSWORD**
+      </td>
+
+      <td>
+        The password for the JMX trust store.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **TIMEOUT**
+      </td>
+
+      <td>
+        The timeout for individual JMX queries in milliseconds.
+      </td>
+
+      <td>
+        10000
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Configure KafkaOffsetSample collection [#KafkaOffsetSample-collection]
+
+The Kafka integration collects both Metrics and Inventory information. Check the **Applies To** column below to find which settings can be used for each specific collection:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "150px" }}>
+        Setting
+      </th>
+
+      <th>
+        Description
+      </th>
+
+      <th>
+        Default
+      </th>
+
+      <th>
+        Applies To
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        **CLUSTER_NAME**
+      </td>
+
+      <td>
+        User-defined name to uniquely identify the cluster being monitored. **Required**.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ "text-align": "center" }}>
+        M/I
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        **KAFKA_VERSION**
+      </td>
+
+      <td>
+        The version of the Kafka broker you're connecting to, used for setting optimum API versions. It must match -or be lower than- the version from the broker.
+
+        Versions older than 1.0.0 may be missing some features.
+
+        **Note that if the broker binary name is kafka_2.12-2.7.0 the Kafka api version to be used is 2.7.0, the preceding 2.12 is the Scala language version**.
```

Comment:
Passive here but not sure it's fixable without muddying up the info. There is a missing comma after the first instance of kafka binary name, though.